WordPress/Classes/ViewRelated/Views/List/ListTableHeaderView.swift

```diff
@@ -12,6 +12,9 @@ class ListTableHeaderView: UITableViewHeaderFooterView, NibReusable {
 
     // MARK: Properties
 
+    @objc static let reuseIdentifier = defaultReuseID
```

Comment:
Nitpick/request - can a comment be added here indicating `reuseIdentifier` can be removed when the VCs are converted to Swift? It'd be nice to _eventually_ use `defaultReuseID` directly.

Reply:
Noted! Will do.